packages/lib/src/vis/tiles/TiledHeatmapMesh.tsx

```diff
@@ -46,9 +50,11 @@ function TiledHeatmapMesh(props: Props) {
   );
   const visibleBox = getObject3DVisibleBox(ndcToLocalMatrix);
 
-  const bounds = scaleToLayer(visibleBox, baseLayerSize, meshSize);
+  const { xScale, yScale } = useLayerScales(baseLayerSize, meshSize);
```

Comment:
Scales are now computed by a hook outside of `scaleToLayer` (renamed `scaleBox`) to be able to access the `AxisContext` and take the flip in account